app/routes/task_routes.py

```diff
@@ -1 +1,71 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, make_response, abort
```

Comment:
`abort` and `make_response` are imported but not accessed so they should be removed.